internal/messagejournal/messagejournal.go

```diff
@@ -0,0 +1,216 @@
+package messagejournal
+
+import (
+	"database/sql"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/redhatinsights/yggdrasil"
+	"github.com/redhatinsights/yggdrasil/ipc"
+)
+
+const messageJournalTemporaryDropString string = `
```

Comment:
I would add some doc string for this variable. Maybe, the name is little bit misleading... I would say that it is not "...String". I would say that it is "...Command"

Reply:
BTW: this variable is used only in function `New()`. Could it be moved to local variable of this function? Or is there any reason, why it is "package" variable?